src/main/java/org/shanerx/tradeshop/commands/Executor.java

```diff
@@ -114,7 +114,13 @@ public boolean onCommand(CommandSender sender, Command cmd, String label, String
 				}
 				
 				Player pl = (Player) sender;
-				ItemStack itm = pl.getInventory().getItemInMainHand();
+				ItemStack itm = null;
+
+				if (plugin.getAboveMC18())
```

Comment:
Since this getter returns a boolean value, the name ``isAbove18`` might be more appropriate.